### PR TITLE
Add FS Data Engineer Job

### PR DESCRIPTION
**Pull request by dancork**
Screenshot:

image

### CODE REVIEW

**Comment by hpoom** [comment]
:+1:  from me

Only comment would be all `&#10004;` in the file. That is not very markdown

**Comment by hpoom** [comment]
To be honest not sure how we should do escape chars in Markdown maybe `&#10004;` is the right way.

**Comment by dancork** [comment]
@hpoom yep, all the roles seem to have them. i've left it like that for uniformity. looking in to a better long-term solution for this - and for blog posts. maybe some markdown linting.

File: _jobs/full-stack-data-engineer.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+A Full-Stack Data Engineer is responsible for building and maintaining data products which sit on our data pipeline infrastructure to enable the business to gain insight and real-time tools and APIs from every data point we collect.  Some of these products are internal, some may be customer or partner-facing.
+
+You’ll have freedom and the opportunity to have a massive impact. Discover new ways to make revenue, solve unusual problems, and do fascinating things with data. You’ll be responsible for driving a data strategy, building a data infrastructure, and working with multiple teams to make our entire company data-centric. This isn’t about analytics, it’s about building products using data.
+
+This is a senior role — requiring understanding the business as well as working with executive stakeholders and many teams. We are looking for someone with:
+
+&#10004; Significant software engineering experience  
+&#10004; Deep knowledge of one mainstream programming language (C#, Python, Java, Ruby, R)  
+&#10004; Deep familiarity with Javascript and at least one JS framework (Angular, Ember, Om, React)  
+&#10004; Experience with and structured, standardised use of JSON API
```
Comment (Nick-Patrick) [inline _jobs/full-stack-data-engineer.md]: This sounds odd. 

Maybe 'Experience with using structured and standardised JSON API?

**Pull request merged**